ldes-fragmentisers/ldes-fragmentisers-pagination/README.md

```diff
@@ -0,0 +1,35 @@
+# Pagination fragmentation
+
+Pagination fragmentation will create fragments based on the order of arrival of the members.
+This is meant to be a standard way of fragmentation.
+
+## Properties
+
+  ```yaml
+  name: "pagination"
+  config:
+    memberLimit: { Mandatory: member limit > 0 }
+  ```
+
+## Algorithm
+
+1. The fragment to which the member should be added is determined.
+   - The currently open fragment is retrieved from the database.
```

Comment:
Suggestion: I realize we use open in the code as well, but maybe we should switch to mutable :-)